defs.bzl

```diff
@@ -214,7 +212,12 @@ def get_fbgemm_inline_neon_srcs(msvc = False, buck = False):
     intrinsics_srcs = ["src/UtilsNeon.cc"]
 
     # FP16 kernels contain inline assembly and inline assembly syntax for MSVC is different.
-    asm_srcs = ["src/UtilsNeon.cc"]
+    asm_srcs = [
```

Comment:
@omegacoleman I think the issue is here.  In the internal builds (`if buck` returns true), when `ovr_config//cpu:arm64` is true, the asm_srcs are not selected, and so we run into `undefined symbol: kleidiai::gemmkernel_1x1_Neon_fp16_fA0fB0fC0`.